src/css/variablesCustom.css

```diff
@@ -173,6 +173,29 @@
   --button-category-font-weight-hover-custom: 600;
   --button-category-background-color-active-custom: var(--button-category-background-color-hover-custom);
   --button-category-color-active-custom: var(--button-category-color-hover-custom);
+
+  /* footer */
+  --footer-default-boarder-right: none;
```

Comment:
Typo
```suggestion
  --footer-default-border-right: none;
```

Reply:
![image](https://github.com/mits-gossau/web-components-toolbox-steps/assets/11610790/82575705-baec-4a93-a8ca-62a56b28d553)

The variable name has a typo out of the box (I don't know why). I just copied it to be able to override it and it works. After fixing the typo, the variable is wrong and shows me the border again. I will keep it like this, because we have to fix it somewhere else I guess.